backend/views/syson-diagram-interconnection-view/src/main/java/org/eclipse/syson/diagram/interconnection/view/InterconnectionViewJavaServiceProvider.java

```diff
@@ -44,7 +45,8 @@ public List<Class<?>> getServiceClasses(View view) {
                     InterconnectionViewEdgeService.class,
                     InterconnectionViewLabelService.class,
                     InterconnectionViewToolService.class,
-                    UtilService.class);
+                    UtilService.class,
+                    ViewEdgeService.class);
```

Comment:
May be `InterconnectionViewEdgeService` could inherits from `ViewEdgeService` instead. What do you think about?

Reply:
Done